### ISSUE TITLE
Exported image doesn't contain Exif data

### ISSUE THREAD

**Issue by siju-s** [Insta360Develop/CameraSDK-Android#20]
Doesn't contain the Exif data on photos exported through the SDK-Demo app too.

**Comment by Tkorn** [comment]
The exported image contains ExposureMode、ISO、ShutterSpeed、Aperture data, you can check it by exif parser software. Any other exif data do u need?

**Comment by Wolf00Bomber** [comment]
@Tkorn you are right, there is code block which sets the above EXIF data in the `exportPhoto()` method in `ExportUtils` class, but by default when I see the exported image in Insta 360 Studio, I don't see those meta data. My question are as follows:
1. Do we need to enable any flag for the metadata to flow into the exported image?
2. Is the EXIF data set for all Image capture types? I am using a `Normal` and `Normal Pano` types of images capture.

For your reference:
1. Insta 360 Camera Studio version:
![Insta 360 Studio version](https://github.com/Insta360Develop/CameraSDK-Android/assets/2338988/dafb5b29-6111-4d56-8454-0aa3482fad52)

2. Insta 360 Exported Image File Data:
![Meta data](https://github.com/Insta360Develop/CameraSDK-Android/assets/2338988/239c8e5c-0bb2-4ece-9290-0112aa32be7c)
